packages/tasks/src/snippets/python.ts

```diff
@@ -12,6 +12,22 @@ output = query({
     "parameters": {"candidate_labels": ["refund", "legal", "faq"]},
 })`;
 
+export const snippetZeroShotImageClassification = (model: ModelData): string =>
+	`def query(data):
+	with open(data["image_path"], "rb") as f:
+		img = f.read()
+	payload={
+		"parameters": data["parameters"],
+		"inputs": base64.b64encode(img).decode("utf-8")
```

Comment:
Interesting! So the input image is received as binary data, I guess. Is there a way to test and debug these snippets before merging?

Reply:
Yes, you can test the python snippet locally or in Colab

```python
import requests
import base64

API_URL = "https://api-inference.huggingface.co/models/openai/clip-vit-base-patch32"

def query(data):
	with open(data["image_path"], "rb") as f:
		img = f.read()
	payload={
		"parameters": data["parameters"],
		"image":  base64.b64encode(img).decode("utf-8"),
	}
	response = requests.post(API_URL, json=payload)
	return response.json()
 
output = query({
	"image_path": "/content/download.png",
	"parameters": {"candidate_labels": ["cat", "dog", "llama"]},
})
```

for example

Reply:
Oh, of course! Thanks a lot! 🙌